package.json

```diff
@@ -12,7 +12,7 @@
   "homepage": "https://github.com/eclipse-che/che-dashboard",
   "engines": {
     "yarn": ">=1.20.0",
-    "node": ">=12"
+    "node": ">=14"
```

Comment:
why not `>=16`?

Reply:
answered https://github.com/eclipse-che/che-dashboard/pull/480#discussion_r833380337